config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "https://ffd-microsite-staging.apps.cloud.gov/"
+baseurl = "https://ffd-microsite-staging.app.cloud.gov/"
```

Comment:
correct. there's probably a better way to do this where both the staging and master are defined in the same config but as it's set up now they need the full baseurl defined to ensure the static assets load correctly in their respective builds/branches. I'll check in with you directly to talk about some better ways to do this.